addons/mail/static/src/model/model_field.js

```diff
@@ -129,6 +130,12 @@ class ModelField {
          * relation is removed, the related record is automatically deleted.
          */
         this.isCausal = isCausal;
+        /**
+         * Determine whether the field is read only or not. Read only field
+         * can't be updated once they are created. Computed fields can't be
+         * writed on but will be recomputed if needed
```

Comment:
"once they are created" sounds wrong, it's the record that is created not the read-only field.

Reply:
As for the last phrase, I would just say that an exception is made [to the previous rule] for computed fields (when their dependencies change), and related fields (when their inverse changes).